OneCommunityGlobal/HGNRest
# Adds user's ability to check entered password to reveal the weeklySummaryRecipient

# Description
The feature implemented in this PR reveals the user password in the weeklySummaryRecepients table for the convenience of the user only if the user is an authorized user.

## Related PRS (if any):
To test this backend PR you need to checkout the [#2124](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2124) frontend PR.
…

## Main changes explained:
- Authorized emails are hard-coded to avoid any confusions
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Include the user email with owner's access on `userProfileController.js` 
<img width="1155" alt="Screenshot 2024-04-07 at 5 51 44 AM" src="https://github.com/OneCommunityGlobal/HGNRest/assets/101291283/1a9609c4-ddd2-479e-938b-731ac6ec6211">


## Screenshots or videos of changes:
Please refer to the video in the frontend code

## Note:
Please note this feature can only be accessed by authorized users, hence please include the email that has OWNER rights both in the Frontend and Backend code .
